lib/svg-sprite/utils/index.js

```diff
@@ -41,6 +41,24 @@ function isString(value) {
   return Object.prototype.toString.call(value) === '[object String]';
 }
 
+/**
+ * Trim the start of a string of the specified characters.
+ *
+ * @param {string} inputString The string to trim the start of.
+ * @param {string} [charsToTrim=" "] The characters to trim from the start of the string. Defaults to a single space. The order of the characters does not matter.
+ * @returns {string} The trimmed string.
+ */
+function trimStart(inputString, charsToTrim = " ") {
+  if (inputString === undefined || inputString === null) {
```

Comment:
I would simplify this as `if (!inputString)`

Reply:
Done; it doesn't match the original API, but it will handle some edge cases